src/main/kotlin/org/rust/ide/presentation/Utils.kt

```diff
@@ -74,6 +74,9 @@ private fun presentableName(psi: RsElement): String? {
             val trait = psi.traitRef?.text
             buildString {
                 if (trait != null) {
+                    if (psi.excl != null) {
```

Comment:
```suggestion
                    if (psi.isNegativeImpl) {
```
This would be more clear